.github/workflows/bump-formula-pr.yml

```diff
@@ -5,9 +5,7 @@ on:
   push:
     tags:
       - 'v*'
-    tags-ignore:
-      # Ignore release candidates
-      - 'v*-rc.*'
```

Comment:
We were [getting some errors](https://github.com/grafana/agent/actions/workflows/bump-formula-pr.yml) about setting both tags and tags-ignore so I'm sneaking in the fix here